packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -325,13 +350,7 @@ class StatefulGraphChatListClient implements StatefulClient<GraphChatListClient>
         // emit new state;
         if (this.userId) {
           void this.updateUserSubscription();
-
-          if (this._graph !== undefined) {
-            loadChatThreads(this._graph, this.chatThreadsPerPage).then(
-              chats => this.handleChatThreads(chats),
-              err => error(err)
-            );
-          }
+          this.loadAndAppendChatThreads('', [], this.chatThreadsPerPage);
         }
         return;
       case ProviderState.SignedOut:
```

Comment:
If we do a sign-out, shouldn't we clear the chat list also?

Reply:
if we do a sign out, we should get redirected out, the entire component should be refreshed.